### DISCUSSION TITLE
More optional shortcuts (open / hide from anywhere)

### DISCUSSION THREAD

**Discussion by nick-wilkins** [spatie/ray#48] [Ideas]
Cmd + Shift + ? could be a bit limiting. Allow more options here, or have the ability to disable completely

**Comment by freekmurze** [comment]
We'll add this in a future update for sure!

**Comment by axelitus** [comment]
I do not use the shortcuts in Ray and it hijacked the shortcuts and conflicts with the ones I have set up in Sublime Text and PhpStorm. As Ray registers the keyboards globally, I wasn't able to use my shortcuts on the other apps. Took me a little while to figure out why the heck my shortcuts weren't working.

Clearing the characters in the shortcuts seems to do the trick, but it would be better if there were a global "Enable/Disable Shortcuts switch".
